src/jobs/docker-build.yml

```diff
@@ -15,4 +23,8 @@ steps:
   - setup_remote_docker
   - run:
       name: Build Dockerfile
-      command: docker build .
+      command: |
+        docker build \
+        --build-arg GITHUB_USERNAME=${<<parameters.github-username>>} \
+        --build-arg GITHUB_TOKEN=${<<parameters.github-token>>} \
```

Comment:
Not sure what the repercussions would be, but passing the token like this would show it in the job build log in CircleCI

Reply:
circleci env variables are encrypted and their value is not shown into logs even if you explicitly try to "echo" them